exporter/diagnostic_data_collector_test.go

```diff
@@ -108,6 +114,23 @@ func TestDiagnosticDataCollectorWithCompatibleMode(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func getVendor(t *testing.T, client *mongo.Client, ctx context.Context, logger *logrus.Logger) (string, error) {
+	buildInfoCmd := bson.D{bson.E{Key: "buildInfo", Value: 1}}
```

Comment:
It makes no sense to execute same logic as one being tested to compare output. Let's map containers to expected output and use that for returning vendor.

Reply:
do you know how can we get the image name in the runtime?

Reply:
Never mind, found how to do that

please check